app/environments/brassbirmingham/brassbirmingham/envs/classes/board.py

```diff
@@ -521,16 +521,22 @@ def getVictoryPoints(self) -> Dict[str, int]:
             if building.isSold:
                 points[building.owner.id] += building.victoryPointsGained
 
-        for roadLocation in ROAD_LOCATIONS:
+        for roadLocation in self.roadLocations:
             if roadLocation.road:
-                print("HERE")
+                print(roadLocation)
+                print(roadLocation.road)
+                print('HERE')
                 # print(roadLocation)
                 roadOwner = roadLocation.road.owner
-                for network in roadLocation.networks:
-                    if network.type == "TradePost":
-                        print(network.networkPoints)
-                        points[roadOwner.id] += network.networkPoints
-                    elif network.type == "Town":
-                        points[roadOwner.id] += network.getNetworkVictoryPoints()
+                for town in self.towns:
```

Comment:
Won't looping through `self.roadLocations` already iterate through all the possible locations that a road providing network VPs could exist at? So there's no need for this town loop as well?

Reply:
Yes that's right